src/BlueSCSI_initiator.cpp

```diff
@@ -399,9 +411,12 @@ void scsiInitiatorMainLoop()
             g_initiator_state.target_file.flush();
 
             int speed_kbps = numtoread * g_initiator_state.sectorsize / (millis() - time_start);
-            log_f("SCSI read succeeded, sectors done: %d / %d speed %d kB/s - %.2f%%",
```

Comment:
Hey @erichelgeson @andrewocyeah, I was looking at this commit and see that this SCSI log line is changed. In the new version it is specific to the details of audio_mode tape transfers. I suggest this should be a conditional, so if audio_mode then use the new log line, else stay with the original.